src/java/picard/sam/markduplicates/EstimateLibraryComplexity.java

```diff
@@ -127,13 +127,21 @@
             "group size would be approximately 10 reads.")
     public int MAX_GROUP_RATIO = 500;
 
+    @Option(doc = "Barcode SAM tag (ex. BC)", optional=true)
```

Comment:
I would also say that an email to ```samtools-devel@lists.sourceforge.net``` requesting that the new tags (```BX```, ```RX```, etc.) be added to the spec early on will help avoid 10X Genomics data getting into the wild using non-standard non-local SAM tags.   See footnote 3: https://samtools.github.io/hts-specs/SAMv1.pdf